docs/operations/faq/zone_aware_routing.rst

```diff
@@ -0,0 +1,53 @@
+.. _common_configuration_zone_aware_routing:
+
+Zone aware routing
+==================
+
+There are several steps required for enabling :ref:`zone aware routing <arch_overview_load_balancing_zone_aware_routing>`
+between source service A and destination service B. See below for specific configuration/setup on source
+and destination service/cluster.
```

Comment:
nit: you can remove cluster. Keeping the terminology consistent is helpful for consumers.